### ISSUE TITLE
Cache Management Creates Uneccesary Clients

### ISSUE THREAD

**Issue by afroDC** [GluuFederation/cluster-mgr#125]
root@c7:~# cat /etc/stunnel/stunnel.conf
cert = /etc/stunnel/cert.pem
pid = /var/run/stunnel.pid
output = /var/log/stunnel4/stunnel.log
[client1]
client = yes
accept = 127.0.0.1:7001
connect = 165.227.187.255:7777
[client2]
client = yes
accept = 127.0.0.1:7002
connect = 138.197.65.243:7777
[redis-server]
client = no
accept = 138.197.108.18:7777
connect = 127.0.0.1:6379
[twemproxy]
client = yes
accept = 127.0.0.1:7000
connect = 127.0.1.1:8888

Everything but `[redis-server]` and `[twemproxy]` are unnecessary addtions. I believe the code here creates this and can be removed: 
https://github.com/GluuFederation/cluster-mgr/blob/master/clustermgr/tasks/cache.py#L285

```
        for s in servers:
            port = 7000 + s.id
            redis_instances.append('localhost:{0}'.format(port))
            stunnel_conf.append("[client{0}]".format(s.id))
            stunnel_conf.append("client = yes")
            stunnel_conf.append("accept = 127.0.0.1:{0}".format(port))
            stunnel_conf.append("connect = {0}:7777".format(s.ip))
```

**Comment by afroDC** [comment]
The only server this needs to happen on is the Proxy server

**Comment by mbaser** [comment]
closed by commit 981551b
